iree/tools/iree-benchmark-module-main.cc

```diff
@@ -64,6 +68,17 @@ static void BenchmarkFunction(
   IREE_TRACE_SCOPE_DYNAMIC(benchmark_name.c_str());
   IREE_TRACE_FRAME_MARK();
 
+  // Warmup.
+  for (int i = 0; i < absl::GetFlag(FLAGS_warmup_iterations); ++i) {
+    IREE_TRACE_SCOPE0("WarmupIteration-" + i);
```

Comment:
the operator+ here seems like it's adding `i` as an offset into that literal string - not what you wanted?

Reply:
ah missed `std:to_string(i)` here =) thanks!

Reply:
this won't work (the strings must be constant literals in read-only memory, not stack-allocated)